<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>525</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="showStreamLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>571</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color: #000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStarteStream">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>159</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Starte Stream</string>
    </property>
   </widget>
   <widget class="QLabel" name="showTrackingLabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>571</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color: #000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>390</y>
      <width>52</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Cord X:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Cord Y:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>390</y>
      <width>245</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="buttonStartTracking">
       <property name="text">
        <string>Starte Tracking</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="buttonStartDrawRoute">
       <property name="text">
        <string>Enter Route</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="buttonResetWindow">
       <property name="text">
        <string>Reset Window</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Platzhalter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1202</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHauptmenue">
    <property name="title">
     <string>Einstellungen</string>
    </property>
    <addaction name="actionIP_EInstellungen"/>
    <addaction name="actionKamerakalibrierung"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="action_ber_EagleEye"/>
   </widget>
   <addaction name="menuHauptmenue"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionIP_EInstellungen">
   <property name="text">
    <string>IP-EInstellungen</string>
   </property>
  </action>
  <action name="actionKamerakalibrierung">
   <property name="text">
    <string>Kamerakalibrierung</string>
   </property>
  </action>
  <action name="action_ber_EagleEye">
   <property name="text">
    <string>Ãœber EagleEye</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
